src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+function getPeopleWithParents(people: Person[]) {
+  const copyPeople = [...people];
+
+  for (const copyPerson of copyPeople) {
+    people.forEach(person => {
+      if (person.name === copyPerson.fatherName) {
+        copyPerson.father = person;
+      }
+
+      if (person.name === copyPerson.motherName) {
+        copyPerson.mother = person;
+      }
+    });
+  }
```

Comment:
```suggestion
  for (const { mother, father, fatherName, motherName  } of copyPeople) {
    people.forEach(({ name }) => {
      if (name === fatherName) {
        father = person;
      }

      if (name === motherName) {
        mother = person;
      }
    });
  }
```

Reply:
can't change `mother` and `father`, they are constants.